acceptance/current/.kitchen.yml

```diff
@@ -23,12 +23,12 @@ platforms:
   # - name: macosx-10.10
   #   driver:
   #     box: chef/macosx-10.10 # private
-  # - name: windows-7-professional
-  #   provisioner:
-  #     name: windows_chef_zero
-  #     require_chef_omnibus: 11.12.4
-  #   driver:
-  #     box: chef/windows-7-professional # private
+  - name: windows-7-professional
```

Comment:
After chatting with some of our Windows team they recommended standardizing on Windows Server 2012R2 for our default Windows test platform. Take a look at:
https://github.com/chef-cookbooks/omnibus/blob/master/.kitchen.yml#L57-L59